### PR TITLE
Add AppVeyor CI

### PR DESCRIPTION
**Pull request by zoidbergwill**

### CODE REVIEW

File: data/tools/appveyor.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  {
+    "slug": "appveyor",
+    "name": "AppVeyor",
+    "description": "Continuous Delivery service for Windows",
+    "url": "https://www.appveyor.com/",
+    "tags": [
+      "windows", 
+      "free",
+      "commercial",
+      "ci",
```
Comment (rdsubhas) [inline data/tools/appveyor.json]: lint error, dangling comma :)

Comment (zoidbergwill) [inline data/tools/appveyor.json]: Awesome, thanks, my bad. Fixing.

**Comment by rdsubhas** [comment]
Thx merging

**Comment by zoidbergwill** [comment]
Awesome, thank you.

**Pull request merged**